INFO 2024-02-05 12:27:52,844 strategy 35491 6210859008 Traceback (most recent call last):
  File "/Users/sid/Desktop/algo-trading/PROFESSIONAL/BASE_DJANGO/datamanagement/helpful_scripts/strategy.py", line 204, in run
    self.main()
  File "/Users/sid/Desktop/algo-trading/PROFESSIONAL/BASE_DJANGO/datamanagement/helpful_scripts/strategy.py", line 185, in main
    df=self.download_ohlc(ticker)
       ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sid/Desktop/algo-trading/PROFESSIONAL/BASE_DJANGO/datamanagement/helpful_scripts/strategy.py", line 97, in download_ohlc
    data=self.client.fetch_ohlcv(instrument,timeframe=self.admin['time_frame'])
         ^^^^^^^^^^^
AttributeError: 'run_strategy' object has no attribute 'client'

INFO 2024-02-05 12:27:57,329 strategy 35507 6203731968 Traceback (most recent call last):
  File "/Users/sid/Desktop/algo-trading/PROFESSIONAL/BASE_DJANGO/datamanagement/helpful_scripts/strategy.py", line 204, in run
    self.main()
  File "/Users/sid/Desktop/algo-trading/PROFESSIONAL/BASE_DJANGO/datamanagement/helpful_scripts/strategy.py", line 185, in main
    df=self.download_ohlc(ticker)
       ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/sid/Desktop/algo-trading/PROFESSIONAL/BASE_DJANGO/datamanagement/helpful_scripts/strategy.py", line 97, in download_ohlc
    data=self.client.fetch_ohlcv(instrument,timeframe=self.admin['time_frame'])
         ^^^^^^^^^^^
AttributeError: 'run_strategy' object has no attribute 'client'

